<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DynamicGroupDialog_q</class>
 <widget class="QWidget" name="DynamicGroupDialog_q">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Match Criteria:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="criteriaView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add Match</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Matching Objects:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ObjectListView" name="matchedView">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CommentKeywords" name="commentKeywords" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>commentKeywords</zorder>
  <zorder>frame</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommentKeywords</class>
   <extends>QWidget</extends>
   <header>CommentKeywords.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ObjectListView</class>
   <extends>QTreeWidget</extends>
   <header>ObjectListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>DynamicGroupDialog_q</receiver>
   <slot>addMatchClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
